packages/core/headings.scss

```diff
@@ -1,28 +1,37 @@
 @import "./variables/font-size";
-@import "./variables/spacing";
-@import "./mixins/helpers";
 
-.jkl-h1 {
-    @include h1-heading;
-    margin-bottom: $layout-spacing--small;
+.jkl-title-large {
+    @include jkl-text-style("desktop/title-large");
+
+    @include small-device {
+        @include jkl-text-style("compact/title-large");
+    }
 }
+.jkl-title-small {
+    @include jkl-text-style("desktop/title-small");
 
-.jkl-h2 {
-    @include h2-heading;
-    margin-bottom: $layout-spacing--small;
+    @include small-device {
```

Comment:
Burde `small-device` implisitt wrappe "compact/*"?

Eksempel: 

```
$text-styles: (
 "compact/title-small": (
	@include small-device {
        font-size: $font-size-9,
        line-height: $line-height-5,
        font-weight: $normal-weight,
	}
  )
)
```

Jeg kan se for meg at dette vil være en problematisk løsning om man noen gang vil bruke "compact" på en ikke-compact enhet, men vil det noen gang inntreffe?

Reply:
Never mind, ser det er beskrevet i dokumentasjonen 😊👍